homeassistant/components/twinkly/light.py

```diff
@@ -163,6 +163,7 @@ def effect_list(self) -> list[str]:
         effect_list = []
         for movie in self._movies:
             effect_list.append(f"{movie['id']} {movie['name']}")
+        effect_list.append('playlist')
```

Comment:
This also needs handling in the `effect` property above.

Reply:
I'm not sure that it does, because a playlist isn't an effect, it is all effects. So if I have a playlist of 2 effects, "Snow" and "Twinkle", when playing the playlist, the current effect is either "Snow" or "Twinkle", not "playlist".